specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-06-02-preview/fleets.json

```diff
@@ -511,6 +511,48 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/Fleets/{fleetName}/listClusterCredential":{
+      "post": {
```

Comment:
since it's an action in ARM concept, similar to list stroageAccountKey 
https://github.com/Azure/azure-rest-api-specs/blob/c537f5a7853766a7946863dd9b0303773b488eff/specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json#L513